docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -283,3 +283,36 @@ function _su_humsci_profile_update_fix_menuposition_sections(array $sections) {
   }
   return $changed;
 }
+
+/**
+ * Uninstall material admin theme and theme switcher module.
+ */
+function su_humsci_profile_update_8012() {
+  $config_factory = \Drupal::configFactory();
+
+  // Delete the old block configurations for the admin theme so that when the
+  // new theme is enabled, the UUIDs will still work and be the same.
+  $query = \Drupal::database()
+    ->select('config', 'c')
+    ->fields('c', ['name'])
+    ->condition('name', 'block.block.su_humsci_admin%', 'LIKE')
+    ->execute();
+
+  // The status region no longer exists, so move any blocks into the help
+  // region.
+  while ($config_name = $query->fetchField()) {
+    $config = $config_factory->getEditable($config_name);
+    if ($config->get('region') == 'status') {
+      $config->set('region', 'help')->save();
+    }
+  }
+
+  // Disable theme switcher module since we no longer need it.
+  \Drupal::service('module_installer')->uninstall(['role_theme_switcher']);
+  \Drupal::service('theme_installer')->uninstall(['material_admin']);
+
+  // Remove the role that is no longer necessary.
+  if ($role = Role::load('seven_admin_theme_user')) {
+    $role->delete();
```

Comment:
Does this also remove the roles from user entities? Looking at the class it does not appear to: https://api.drupal.org/api/drupal/core%21modules%21user%21src%21Entity%21Role.php/class/Role/8.7.x

Reply:
It does in some way. Upon deleting the role if i load a user entity, the user does not have the `seven_admin_theme_user` role. To double check, i manually created the role again. the user still did not have the role.

Reply:
This is great. Thank you for checking that out.